user/templates/debian/debian.md

```diff
@@ -11,11 +11,20 @@ ref: 134
 title: Debian templates
 ---
 
-The Debian [template](/doc/templates/) is an officially [supported](/doc/supported-releases/#templates) template in Qubes OS.
-This page is about the standard (or "full") Debian template.
-For the minimal version, please see the [Minimal templates](/doc/templates/minimal/) page.
+The Debian [template](/doc/templates/) is an officially [supported](/doc/supported-releases/#templates) template in Qubes OS.  
+This page is about the standard (or "Full") Debian template.
+For the Minimal version, please see the [Minimal templates](/doc/templates/minimal/) page.
 There is also a [Qubes page on the Debian Wiki](https://wiki.debian.org/Qubes).
 
+
+| Release Version        |  Initial Release Date & Status*       |  "Full" Qubes OS template available? |  "Minimal" Qubes OS template available? |
+| :---                   |  :---                                 |  :---                                                |  :---                                                  |
+| Debian 11 "bullseye"   | 14 Aug 2021, current stable           |  —Not Yet—                                           |  —Not Yet—                                                     |
+| Debian 10 "buster"     | 06 Jul 2019, current oldstable            |  Yes                                             |  Yes                                                    |
+| Debian 9 "stretch"     | 27 Jun 2017, oldoldstable under LTS       |  Decomissioned                                   | Decomissioned                                           |
+| Debian 8 "jessie"      | 26 Apr 2015, archived under Extended LTS  |  Decomissioned                                   | Decomissioned                                           |
+| Debian 7 "wheezy"      | 04 May 2013, obsolete stable              |  Decomissioned                                   | Decomissioned
+
```

Comment:
This appears to largely duplicate the table we already have on the "supported releases" page. I propose we just link to that existing table instead. (Note that there's also Fedora to consider.)

Reply:
It feels like a contradiction of our own consistency standard to speak to 3rd party products, on a Qubes docs page on "Supported Releases." That page has enough about just Qubes and dom0 on it, already—and the rest of its contents I feel would make more sense on the Templates page, and the Debian and Fedora specific pages. See other comment. Can update PR, accordingly if you'd be open to that.